lib/src/main/java/graphql/nadel/engine/transform/hydration/batch/NadelBatchHydrator.kt

```diff
@@ -115,6 +119,7 @@ internal class NadelBatchHydrator(
         state: State,
         instruction: NadelBatchHydrationFieldInstruction,
         parentNodes: List<JsonNode>,
+        hints: NadelExecutionHints,
```

Comment:
Couldn't we also just pull it out of `state.executionContext.hints`? Would be easier to clean up later if we avoid passing extra parameters